### PR TITLE
bank-account: split description into instructions and introduction

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/bank-account/instructions.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Instructions
+
+Your task is to simulate a bank supporting opening/closing accounts, withdrawals, and deposits of money.
```
Comment (BNAndras) [inline exercises/bank-account/instructions.md:3]: This sentence reads to me like our code is responsible for managing the bank itself, but in reality, the exercise is modeling an account inside the bank.

File: exercises/bank-account/instructions.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+As bank accounts can be accessed in many different ways (internet, mobile phones, automatic charges), make sure that your bank accounts can safely be accessed from multiple threads/processes (terminology depends on your programming language) in parallel.
```
Comment (IsaacG) [inline exercises/bank-account/instructions.md:5]: Is it worth being a bit more verbose here and spelling out what needs doing in more detail?

Comment (ErikSchierboom) [reply exercises/bank-account/instructions.md:5]: Maybe so. Thoughts on wording?

Comment (IsaacG) [reply exercises/bank-account/instructions.md:5]:
```suggestion
As bank accounts can be accessed in many different ways (internet, mobile phones, automatic charges), your bank software must allow accounts to be safely accessed from multiple threads/processes (terminology depends on your programming language) in parallel.
For example, there may be many deposits and withdrawals occurring in parallel; you need to ensure there is no [race conditions][helpful link] between when you read the account balance and set the new balance.
```

**Review by BNAndras — APPROVED** [review]

File: exercises/bank-account/instructions.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Instructions
+
+Your task is to implement bank accounts supporting opening/closing, withdrawals, and deposits of money.
+
+As bank accounts can be accessed in many different ways (internet, mobile phones, automatic charges), your bank software must allow accounts to be safely accessed from multiple threads/processes (terminology depends on your programming language) in parallel.
+For example, there may be many deposits and withdrawals occurring in parallel; you need to ensure there is no [race conditions][helpful link] between when you read the account balance and set the new balance.
```
Comment (glennj) [inline exercises/bank-account/instructions.md:6]: `you need to ensure there is no [race conditions][helpful link]`

Does anyone have a helpful link for race conditions?

* https://en.wikipedia.org/wiki/Race_condition#In_software
* https://www.techtarget.com/searchstorage/definition/race-condition
* https://devopedia.org/race-condition-software

Comment (BethanyG) [reply exercises/bank-account/instructions.md:6]: I liked this one for its refreshing lack of ads: https://startup-house.com/glossary/race-condition.
But the devopedia one is nice with its diagrams.

Comment (ErikSchierboom) [reply exercises/bank-account/instructions.md]: I like the wikipedia link best, with clear text and the table it shows is very helpful I feel.

**Review by glennj — APPROVED** [review]

**Review by IsaacG — APPROVED** [review]

**Review by BethanyG — APPROVED** [review]

**Pull request merged**